### PR TITLE
Remove Map / Update BCP-47 Libraries

### PR DESCRIPTION
**Pull request by ShiningTrapez**

### CODE REVIEW

File: src/streaming/models/VideoModel.js [hunk 1]
```diff
@@ -36,13 +36,16 @@ import Debug from '../../core/Debug';
 import Constants from '../constants/Constants';
 import Settings from '../../core/Settings';
 
+const READY_STATES_TO_EVENT_NAMES = function () {
```
Comment (eirikbjornr) [inline src/streaming/models/VideoModel.js:39]: What's the reason this is an IIFE?

**Review by eirikbjornr — APPROVED** [review]

File: src/streaming/models/VideoModel.js (hunk 1 above)
Comment (VriskaBitterstar) [reply src/streaming/models/VideoModel.js:39]: More like how the Map Constructor works IMO, didn't like having a bunch of assigns at the top level

Comment (eirikbjornr) [reply src/streaming/models/VideoModel.js:39]: Ohh is dynamically resolved keys an ES6 feature?

Comment (VriskaBitterstar) [reply src/streaming/models/VideoModel.js:39]: I don't know but it didn't work when I tried it

**Pull request merged**